integration/package.json

```diff
@@ -10,21 +10,23 @@
     "dev": "lerna run test --scope integration --parallel --include-filtered-dependencies"
   },
   "dependencies": {
-    "@shapeshiftoss/hdwallet-core": "^1.14.2",
-    "@shapeshiftoss/hdwallet-keepkey": "^1.14.2",
-    "@shapeshiftoss/hdwallet-keepkey-nodewebusb": "^1.14.2",
-    "@shapeshiftoss/hdwallet-keepkey-tcp": "^1.14.2",
-    "@shapeshiftoss/hdwallet-trezor": "^1.14.2",
-    "debug": "^4.1.1",
+    "@shapeshiftoss/hdwallet-core": "^1.15.0",
+    "@shapeshiftoss/hdwallet-keepkey": "^1.15.0",
+    "@shapeshiftoss/hdwallet-keepkey-nodewebusb": "^1.15.0",
+    "@shapeshiftoss/hdwallet-keepkey-tcp": "^1.15.0",
+    "@shapeshiftoss/hdwallet-trezor": "^1.15.0",
+    "@types/debug": "^4.1.5",
+    "@types/jest": "^26.0.23",
+    "debug": "^4.2.0",
     "fast-json-stable-stringify": "^2.1.0",
+    "jest": "^26.6.3",
```

Comment:
Seems  like some weird dependency inclusions here. Can we make use the of the global dev dependencies for the likes of jest/lerna/typescript etc, and then pull out any non required dependencies into devDependencies?

Reply:
The integration tests need all this stuff to actually run, so some things you'd expect to be in `devDependencies` are actually in `dependencies`. (Not that anyone's probably going to install the integration test package by itself, but if they did, they'd need `jest` to run the package, not just dev on it.)